.github/workflows/install.yml

```diff
@@ -65,29 +68,32 @@ jobs:
           - windows-latest
           - macos-latest
         python-version:
-          - "3.8"
           - "3.9"
           - "3.10"
           - "3.11"
+          - "3.12"
         pytorch-computation-backend:
           - cpu
-          - cu117
           - cu118
+          - cu121
+          - cu124
```

Comment:
The matrix of CI workflows is already massive. I'd keep it to a single CUDA 11 and 12.

Reply:
So just cu118 and cu124? I do recognize the matrix of CI workflows is huge and was thinking about how to best to optimize that in a future PR. 

One specific reason have cu118, cu121 and cu124 here is because Pytorch 2.5.x specifies these 3 options of stable CUDA to pick from. I know earlier pytorch versions that typically had 2 or even 1 cuda compute whl to select. Unclear why pytorch developers kept the cu121 option with cu124 being available as well, but I figured I would keep the same level of support in testing for this repository as well.

https://github.com/pytorch/pytorch/blob/91bf2e16debdc41f5dde2bb5cc8e4f39f8955d4e/RELEASE.md#release-compatibility-matrix
https://pytorch.org/